src/App.js

```diff
@@ -1,26 +1,32 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import data from './api/movies.json';
 
-export class App extends Component {
-  state = {
-    movies: data,
+export const App = () => {
+  const [movies, setMovie] = useState(data);
+  const [addErrorMessage, getAddError] = useState(true);
```

Comment:
```suggestion
  const [addErrorMessage, setAddError] = useState(true);
```
set - встановлювати. get - діставати. Коли ти щось записуєш в стейт, використовується неймінг з set

Reply:
тут с тем посылом, что get - получить сообщение об ошибке

Reply:
Но эта функця не получает сообщение об ошибке, она записывает его в state